spec/facades/giphy_facade_spec.rb

```diff
@@ -21,21 +21,21 @@
       it "returns a sample from of an 'R' rated gif if service response has one" do
         nsfw_response = GiphyFacade.mean_or_nice("nsfw")
         
-        expect(nsfw_response[0][:rating]).to_not eq(("g").to_json)
-        expect(nsfw_response[0][:rating]).to_not eq(("pg-13").to_json)
-        expect(nsfw_response[0][:rating]).to_not eq(("pg").to_json)
+        expect(nsfw_response[0][:rating]).to_not eq("g")
```

Comment:
Is this overkill? If we are testing that the rating is 'R' explicitly, aren't we implicitly testing that it's not 'g', 'pg', 'pg-13' ?